frontend/test/pytest/test_jvpvjp.py

```diff
@@ -832,5 +832,111 @@ def J_workflow():
         assert_allclose(r_j, r_c)
 
 
+@pytest.mark.parametrize("diff_method", diff_methods)
+def test_jvpvjp_argument_type_checks(diff_method):
```

Comment:
Could separate the vjp/jvp in different tests and checks one error per test?

Reply:
Will do